tests/integration/models/billing.spec.coffee

```diff
@@ -143,25 +143,34 @@ describe 'Billing Model', ->
 				resin.models.billing.getPlan()
 				.then (plan) ->
 					m.chai.expect(plan).to.deep.match
-						title: 'TestDev'
-						name: 'TestDev plan'
-						code: 'testdev'
-						tier: 'testdev'
+						title: 'Team member'
+						name: 'Team member plan'
+						code: 'free'
```

Comment:
Is it really free?

The API definition doesn't have a code https://github.com/resin-io/resin-api/blob/master/src/utils/definitions/billing-plans-definition.ts#L148

Reply:
I guess that it's filled as a default value but you are right this does seem strange.

Reply:
This does seem odd. Something to look at for the API rather than here though.